Cargo.lock

```diff
@@ -5082,7 +5082,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "97fee6b57c6a41524a810daee9286c02d7752c4253064d0b05472833a438f675"
 dependencies = [
  "cfg-if",
- "rand 0.8.5",
+ "rand 0.7.3",
```

Comment:
this is a bit surprising, but /shrug

Reply:
I think it's not the first time I've seen it randomly switch between these two versions 😅